client/i18n/locales/english/intro.json

```
@@ -1655,47 +1655,69 @@
       },
       "learn-how-to-use-modal-verbs": {
         "title": "Learn How to Use Modal Verbs",
-        "intro": ["", ""]
+        "intro": [
+          "In this course, you will learn how to use words like must, have to, might, and could to talk about rules, what's necessary, and what could happen.You'll also practice asking and answering questions clearly, and adding helpful details to your ideas."
```

```suggestion
          "In this course, you will learn how to use words like `must`, `have to`, `might`, and `could` to talk about rules, what's necessary, and what could happen. You'll also practice asking and answering questions clearly, and adding helpful details to your ideas."
```
We didn't add backticks to EC intros because there was no need, but I think it's better to add them here. They're often used for CFSD

---

@anastasiiauk The backticks won't work on this kind of file. I've tried but as far as I know they don't.